cmd/wego/app/add/cmd.go

```diff
@@ -30,10 +30,19 @@ var Cmd = &cobra.Command{
 	Use:   "add [--name <name>] [--url <url>] [--branch <branch>] [--path <path within repository>] [--private-key <keyfile>] <repository directory>",
 	Short: "Add a workload repository to a wego cluster",
 	Long: strings.TrimSpace(dedent.Dedent(`
-        Associates an additional application in a git repository with a wego cluster so that its contents may be managed via GitOps
-    `)),
-	Example:       "wego app add .",
-	RunE:          runCmd,
+		Associates an additional application in a git repository with a wego cluster so that its contents may be managed via GitOps
+	`)),
+	Example: `
+  # Add podinfo application to wego control from local git repository
+  wego app add .
+
+  # Add podinfo application to wego control from github repository
+  wego app add github.com/myorg/podinfo
+
+  # Get status of podinfo application
+  wego app status podinfo
+`,
+	RunE:    runCmd,
 	SilenceUsage:  true,
 	SilenceErrors: true,
 	PostRun: func(cmd *cobra.Command, args []string) {
```

Comment:
Below in the code we add the `name` parameter and the help say `Name of remote git repository`. This needs to be changed to `Name of application`